src/qt/guiutil.cpp

```diff
@@ -70,10 +73,26 @@ namespace GUIUtil {
 
 QString loadStyleSheet(){
     QFile file(":/css/main");
+    QFile fileDark(":/css/main-dark");
+    QString stylesheet = QString();
     if(file.open(QFile::ReadOnly)){
-        return QLatin1String(file.readAll());
+       	stylesheet += QLatin1String(file.readAll());
+    }
+
+    if (isDarkModeSet == 0){
+    	 QSettings settings;
+    	 isDarkModeSet = settings.value("bDarkMode", false).toBool();
     }
-    return QString();
+
+    if(fileDark.open(QFile::ReadOnly) && isDarkModeSet){
```

Comment:
nit: please use consistent style for spacing and indentation here. I prefer space after `if` and before `{` and 4-space indents (Github seems to round tabs to 4 or 8 spaces depending on space before them).

Reply:
I will likely not be getting to this PR anytime soon but commenting as noted.